src/actions/service.js

```diff
@@ -0,0 +1,29 @@
+import actionTypes from '../constants/actions';
+import serviceAPI from '../utils/api/service';
+
+export const pricesRetrieved = () => (dispatch, getState) => {
+  const { settings: { token } } = getState();
+
+  serviceAPI.getPriceTicker(token.active)
+    .then(priceTicker => dispatch({
+      type: actionTypes.pricesRetrieved,
+      data: {
+        priceTicker,
+        activeToken: token.active,
+      },
+    }))
+    // eslint-disable-next-line no-console
+    .catch(err => console.log(err)); // TODO: DISCUSS & HANDLE THIS!
```

Comment:
I propose to handle it with an error toast. But the final call is on @reyraa

Reply:
done, we can use a toast error message